### PR TITLE
{CI} Add resolve issues logic

### PR DESCRIPTION
**Pull request by wangzelin007**
Add [automated workflows](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/490/Resolving-Issues) to align with the .NET SDK repo：

1. When work on an issue is believed to be complete, the Azure SDK team member responsible for communicating that status may apply the issue-addressed label.

2. When the bot that monitors the repository detects the issue-addressed label, it will respond with the message:
“Hi @issue-author. Thank you for opening this issue and giving us the opportunity to assist. We believe that this has been resolved. If you believe that further discussion is needed, please add a comment with the text “/unresolve” to remove the "issue-addressed” label and continue the conversation.”

3. For issues where the issue-addressed label was applied:
    a. If no action is taken, the issue is closed by the bot after 7 days of no activity with the message:
“Hi @issue-author, since you haven’t asked that we “/unresolve” the issue, we’ll close this out. If you believe further discussion is needed, please add a comment “/unresolve” to reopen the issue.”

    b. If the original author of the issue uses the /unresolve command to remove the issue-addressed label, the issue is left open for continued discussion and the “needs-team-attention” label is applied.

    c. If a comment is left with the /unresolve command by a person other than the original author of the issue or an internal team member, the bot responds with the message:
“Please open a new issue with your scenario and details if you would like to discuss this topic with the team.”

    d. If the issue-addressed label is removed by an Azure SDK team member directly, the issue is left open but no additional label is applied.

    e. If an Azure SDK team member adds a label that indicates work is ongoing for the issue, the issue-addressed label is removed. The following labels are considered:
         -  needs-team-attention
         - needs-author-feedback
         - Service Attention
         - needs-triage
         - needs-team-triage

### CODE REVIEW

**Comment by yonzhan** [comment]
CI

**Comment by jiasli** [comment]
Is there a source issue/PR that documents the behavior in the PR description?

File: .github/fabricbot.json [hunk 1]
```diff
@@ -722,6 +722,407 @@
...
+        "eventNames": [
+          "issues",
+          "project_card"
+        ],
+        "taskName": "[Resolve Workflow] Issue Addressed Label Applied",
+        "actions": [
+          {
+            "name": "addReply",
+            "parameters": {
+              "comment": "Hi @${issueAuthor}.  Thank you for opening this issue and giving us the opportunity to assist.  We believe that this has been addressed.  If you feel that further discussion is needed, please add a comment with the text “`/unresolve`” to remove the “issue-addressed” label and continue the conversation."
```
Comment (jiasli) [inline .github/fabricbot.json:752]: I am curious how this can be tested before mering this PR?

Comment (wangzelin007) [reply .github/fabricbot.json:752]: We can't test this feature before merging this PR, but we can add this config in portal and export to json file.
So actually the fabric bot portal will make sure this configuration is correct.

**Review by jsntcy — APPROVED** [review]

**Pull request merged**